src/views/Overview/Overview.tsx

```diff
@@ -91,6 +92,7 @@ export class Overview extends React.Component<{}, any> {
     componentDidMount() {
         this.setTitle();
         notification_manager.event_emitter.on("turn-count", this.setBoardsToMoveOn);
+        data.watch("config.user", (user) => this.setState({"user": user}));
```

Comment:
When we watch something in a component we need to `unwatch` it in `componentWillUnmount` too

Reply:
Done!